java/squeek/appleskin/network/SyncHandler.java

```diff
@@ -3,7 +3,7 @@
 import io.netty.buffer.Unpooled;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
-import net.fabricmc.fabric.api.network.ClientSidePacketRegistry;
+import net.fabricmc.fabric.impl.networking.ClientSidePacketRegistryImpl;
```

Comment:
This looks like the wrong way to update this. The deprecation notice says to migrate to ClientPlayNetworking. See also https://fabricmc.net/wiki/tutorial:networking